titus-api/src/test/java/com/netflix/titus/api/jobmanager/model/job/JobCompatibilityTest.java

```diff
@@ -102,17 +103,29 @@ public void testDisruptionBudgetIsIgnored() {
         assertThat(compatibility.isCompatible()).isTrue();
     }
 
+    @Test
+    public void testEnvIsIgnored() {
+        JobDescriptor<ServiceJobExt> reference = JobDescriptorGenerator.oneTaskServiceJobDescriptor();
+        JobDescriptor<ServiceJobExt> other = reference.toBuilder()
+                .withContainer(reference.getContainer().but(container ->
+                        container.toBuilder().withEnv(copyAndAdd(container.getEnv(), "SOME", "OVERRIDE"))
+                ))
+                .build();
+        JobCompatibility compatibility1 = JobCompatibility.of(reference, other);
```

Comment:
This test only has a simple `JobCompatibility `. Should the variable be just `compatibility`?